src/plugins/MapStitching/MapStitchCollect.vue

```diff
@@ -0,0 +1,62 @@
+<template></template>
+<script>
+
+export default {
+  props: {
+    location: {},
+    settings: {},
+  },
+  subscriptions() {
+    async function hash(string) {
+      const hashBuffer = await crypto.subtle.digest('SHA-256', string);
+      const hashArray = Array.from(new Uint8Array(hashBuffer));
+      const hashHex = hashArray
+        .map((bytes) => bytes.toString(16).padStart(2, '0'))
+        .join('');
+      return hashHex;
+    }
+    var nodeDB = {};
+    const hashLink = this.settings.hashLink
+    const uploadLink = this.settings.uploadLink
```

Comment:
Would you not rather have only one setting for server base URL (i.e. without path)? One setting instead of 2. This also makes it easier to add new endpoints and deprecate old ones in the future.
So user sets it up to be `https://map.soulforged.io` and then in your plugin you do `${baseUrl}/hashCheck`